tests/unit/test_models.py

```diff
@@ -415,9 +440,9 @@ class GrammarValidation(pydantic.BaseModel):
         )
     err = raised.value.errors()
     assert len(err) == 1
-    assert err[0]["loc"] == ("x",)
-    assert err[0]["type"] == "value_error"
-    assert err[0]["msg"] == "Value error, value must be a str: [35]"
+    assert err[0]["loc"] == ("x", 0, 1)
+    assert err[0]["type"] == "string_type"
+    assert err[0]["msg"] == "Input should be a valid string"
```

Comment:
This is now reproducing the pydantic error as if you'd put `[35]` into a `str` field, but with the correct location.